### ISSUE TITLE
[DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated when Browsersync: Connected

### ISSUE THREAD

**Issue by kevinmamaqi** [BrowserSync/browser-sync#1688]
### Issue details

As soon as I connect to the server (localhost:3000) I received the following message:
(node:37943) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated

### Steps to reproduce/test case

Use node, npm and gulp as indicated below. All the tasks load properly, but when connected with browser-sync at localhost:3000 node sends the DEP0066 message and it stops working.

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [  2.26.5  ]
- Node        [  12.1.0  ]
- Npm         [   6.9.0 ]

### Affected platforms

- [ ] linux
- [ ] windows
- [X] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [ ] API
- [X] Gulp
- [ ] Grunt
- [ ] CLI

### for all other use-cases, (gulp, grunt etc), please show us exactly how you're using Browsersync

```
function browsersync(){
  browserSync.init( {
    proxy: projectURL,
    open: false,
    injectChanges: true,
  } );
};

gulp.task( 'default', gulp.series(styles, php, phpDIST, vendorsJS, customJS, images, browsersync, watchTask));
```

**Comment by suhailkc** [comment]
I have the problem

**Comment by kevinmamaqi** [comment]
Hey @suhailkc I had to downgrade to node 10.15.3 LTS to solve it.

**Comment by XhmikosR** [comment]
It's just a warning, so it seems we can ignore it for now. But it'd be nice if this was fixed at some point @shakyShane :)

**Comment by XhmikosR** [comment]
https://nodejs.org/api/deprecations.html#deprecations_dep0066_outgoingmessage_headers_outgoingmessage_headernames

I'd submit a PR, but I'm not familiar with the codebase.

**Comment by hewentaowx** [comment]
I have the same problem，what is the solution?

**Comment by XhmikosR** [comment]
You ignore the warning, or wait, or use Node.js < 12.x.

**Comment by laurentpayot** [comment]
Same on Ubuntu 19.04.

**Comment by hewentaowx** [comment]
thanks！

**Comment by revelt** [comment]
The message does pop up on Node 12 but everything seems to work fine. Does anybody experience any problems on Node 12?

**Comment by revelt** [comment]
for the record, the `DEP0066` message does appear but reloading continues to work, I'm able to use node `12.3.0` to run `browser-sync` `2.26.5`.

<img width="737" alt="Screen Shot 2019-05-26 at 11 08 48" src="https://user-images.githubusercontent.com/8344688/58380270-eda7b300-7fa6-11e9-94c8-afc36bd3df2f.png">

Original post implies "it stops working" but I can see things refreshing on my machine! I'm using Gulp and `browserSync` mostly copy-pasted from official examples, the `.pipe(browserSync.stream());` and all.

I'm not denying existence of your issue, I'm saying maybe something is wired up not the right way if refreshing is not happening?

Either way, I hope the API will be updated soon and `DEP0066` error will disappear. 👍

**Comment by yahacom** [comment]
Same warning on `browserSync 2.26.5` with `node 12.3.1` on windows platform.

**Comment by YJPL** [comment]
Same warning using ```2.26.5``` with node ```12.4.0``` on macOS 10.14.5

**Comment by izulfi** [comment]
just use "browser-sync": "~2.26.7" instead of 2.24.7, in your project. that has fixed warnings for me.

**Comment by suhailkc** [comment]
Update to "browser-sync": "^2.26.7"

**Comment by fedulovivan** [comment]
For someone who came here for solution: 
Issue may be fixed with upgrading [express](https://www.npmjs.com/package/express) package to v4.17.1 (we are on node 12.13.0)

**Comment by ermannos** [comment]
I'm working with express v4.17.1 and node 12.13.1 and for me the problem is still present

**Comment by Arjunalapsapkota** [comment]
Same is the case for me :  express v4.17.1 and node 12.13.1 . What is the solution?

**Comment by nanthan** [comment]
If people come here from Google, you can fix them by upgrade express to version 4.17.1 then an error has gone.
